### PR TITLE
DOC Add instructions for making a release

### PR DESCRIPTION
**Pull request by rth**
This adds some documentation on how to make a release. Please let me know if you have any comments or questions.

I also moved the "Contributing to the “core” C Code" page under the "Contributing" page in the left side bar.

[Rendered version](https://pyodide--1533.org.readthedocs.build/en/1533/development/maintainers.html)

### CODE REVIEW

File: docs/development/contributing.md [hunk 1]
```diff
@@ -137,6 +141,9 @@ If patches fail to apply automatically, one solution can be to
    ```
    git format-patch -<N> -N --no-stat HEAD -o <out_dir>
    ```
+## Maintainer information
+
+For information regaring making releases see {ref}`maintainer-information`.
```
Comment (phorward) [inline docs/development/contributing.md:146]: "regarding to making a release, see"?

File: docs/development/maintainers.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   docker push
+   ```
+   where `BB` is the last version of the `pyodide-env` Docker image.
+7. Revert Step 1. and increment the version in
+   `src/pyodide-py/pyodide/__init__.py` to the next version specified by
+   Semantic Versioning.
+
+### Making a minor release
+
+For the minor release commits need to be added to the `stable` branch, ideally via a PR.
```
Comment (phorward) [inline docs/development/maintainers.md:61]: "For the minor release, commits" <-- just add the comma

**Review by phorward — CHANGES_REQUESTED** [review]
Hello @rth,
thank you very much for writing down this detailed guide on creating a release. I found just two little mistakes, after that we can merge. Maybe some steps could also be put into some kind of shell script later, e.g. updating the contributors list.

**Comment by rth** [comment]
Thanks @phorward, I addressed you comment.

**Review by phorward — APPROVED** [review]

**Pull request merged**